### PR TITLE
Vladimir nimchyn

### PR DESCRIPTION
**Pull request by nima-vova**

### CODE REVIEW

**Comment by nima-vova** [comment]
add twig and controller

**Comment by ahonymous** [comment]
Why overrided file permitions?

File: src/Nima/BlogBundle/Resources/views/Default/baseOneMainDiv.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{%block MainDiv%}
```
Comment (ahonymous) [inline src/Nima/BlogBundle/Resources/views/Default/baseOneMainDiv.html.twig:4]: maybe name of block shoul be as main, not MainDiv

File: src/Nima/BlogBundle/Resources/views/Default/contacts.html.twig [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{% extends 'NimaBlogBundle:Default:mostRead.html.twig' %}
+{% block startMainDiv %}
+    {% use 'NimaBlogBundle:Default:baseOneMainDiv.html.twig' %}
+    {{ parent() }}
+
+{% endblock%}
+{% block left %}
```
Comment (ahonymous) [inline src/Nima/BlogBundle/Resources/views/Default/contacts.html.twig:14]: if don`t uae block in same template - not declare him

Comment (ahonymous) [reply src/Nima/BlogBundle/Resources/views/Default/contacts.html.twig:14]: if him not included in other block

File: src/Nima/BlogBundle/Resources/views/Default/footer.html.twig [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="footer" >
+    ©2016 super puper blog....... ||   twiter....|| facebook ...........|| vk ....|| email: brbr@gmail.com ........................
+</div>
+<!-- Latest compiled and minified JavaScript -->
+<script src="{{asset('js/jquery-3.1.1.min.js')}}"></script>
+<script src="{{asset('js/bootstrap.min.js')}}"></script>
+</body>
```
Comment (ahonymous) [inline src/Nima/BlogBundle/Resources/views/Default/footer.html.twig:7]: tags body and html move to the base template and js move to javascripts block

File: src/Nima/BlogBundle/Resources/views/Default/showPost.html.twig [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+<div class="row"  id="idMainDiv">
+    <div class="row"  >
+        {% include 'NimaBlogBundle:Default:left_menu.html.twig'%}
+
+        <div class="col-md-9 last-post-block" >
+            <span class="label label-default spanMeny">post name: {{ data['title'] }}</span>
+            <div class="lastPost-mini-block">
+
+                <br>
+                {{ data['text_post'] }}
```
Comment (ahonymous) [inline src/Nima/BlogBundle/Resources/views/Default/showPost.html.twig:12]: O_O , it`s array?

**Comment by nima-vova** [comment]
project closed

**Pull request closed without merging**